images/pytorch-pods/launch_k8s_workers.py

```diff
@@ -148,6 +152,15 @@ def main(argv):
     'TPU_NUM_DEVICES': '8',
   }
 
+  if FLAGS.volumes:
+    volumes = {
+      name: mount_path for name, mount_path in
+      [v.split(':') for v in FLAGS.volumes]
+    }
+    print(volumes)
```

Comment:
In some places you use `logging` but here `print`. Just confirm this is what you wanted

Reply:
Oops, that's left over form debugging.